domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -102,7 +108,8 @@ public static Claim claim(ClaimData claimData, String referenceNumber) {
             null,
             null,
             null,
-            null
+            null,
+            new ArrayList<>()
```

Comment:
As we are passing null for many of the constructor arguments, will suggest to use builder rather than calling constructor

Reply:
good one Saba, added now